src/omero_zarr/cli.py

```diff
@@ -259,7 +263,8 @@ def polygons(self, args: argparse.Namespace) -> None:
     def export(self, args: argparse.Namespace) -> None:
         if isinstance(args.object, ImageI):
             image = self._lookup(self.gateway, "Image", args.object.id)
-            if args.bf:
+            print(dir(args))
```

Comment:
I assume this was used for testing and should be removed or turned into logging statement?

Reply:
ah, good catch. thanks.